src/commands/sbt_cached.yml

```diff
@@ -70,7 +70,13 @@ steps:
       steps:
         - run:
             name: Background sbt server
-            command: SBT_NATIVE_CLIENT=false sbt
+            command: |
+              set +eo pipefail
+              SBT_NATIVE_CLIENT=false sbt
+              # if previous command failed create file
+              if [ $? -ne 0 ]; then
+                touch .sbt-failed-to-start
```

Comment:
You could just `exit 1` here on failure to start and avoid the failure detection logic in the next step

Reply:
Failures in `background` jobs don't make the workflow fail in CircleCI.
I found this workaround [here](https://discuss.circleci.com/t/how-do-i-make-a-background-step-fail-the-job/26134/5)

Reply:
Ok, go ahead. It fells a bit hacky, but it gets the job done. I may come back to this if I think of a better solution :)